src/app/containers/CpsRecommendations/index.jsx

```diff
@@ -0,0 +1,204 @@
+import React, { useContext } from 'react';
+import { arrayOf, shape, number, string } from 'prop-types';
+import styled from 'styled-components';
+import path from 'ramda/src/path';
+import SectionLabel from '@bbc/psammead-section-label';
+import StoryPromo, { Headline, Link } from '@bbc/psammead-story-promo';
+import { StoryPromoLi, StoryPromoUl } from '@bbc/psammead-story-promo-list';
+import { C_LUNAR, C_EBON, C_GHOST } from '@bbc/psammead-styles/colours';
+import Image from '@bbc/psammead-image';
+import { getDoublePica } from '@bbc/gel-foundations/typography';
+import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+import {
+  GEL_SPACING,
+  GEL_SPACING_HLF,
+  GEL_SPACING_DBL,
+} from '@bbc/gel-foundations/spacings';
+
+import { ServiceContext } from '#contexts/ServiceContext';
+import useToggle from '#hooks/useToggle';
+import CpsOnwardJourney from '../CpsOnwardJourney';
+import Grid from '../../components/Grid';
+import data from './recommendations.ltr.json';
+
+const GEL_SPACING_DBL_MINUS_HLF = '0.75rem';
+
+const StoryPromoWrapper = styled(StoryPromo)`
+  display: grid;
+  margin: ${GEL_SPACING_HLF} ${GEL_SPACING_DBL};
+  background-color: ${C_GHOST};
+`;
+
+const CustomSectionLabel = styled(SectionLabel).attrs(props => ({
+  'aria-labelledby': props.labelId,
+}))`
+  ${({ script }) => script && getDoublePica(script)};
+  ${({ service }) => getSansRegular(service)};
+  color: ${C_EBON};
+  margin-top: 0;
+  background-color: ${C_LUNAR};
+`;
+
+const SectionLabelWrapper = styled.div`
+  padding: ${GEL_SPACING_DBL};
+`;
+
+const WsojWrapper = styled.div`
+  background-color: ${C_LUNAR};
+  padding-bottom: ${GEL_SPACING};
+`;
+
+const StyledStoryPromoLi = styled(StoryPromoLi)`
+  /* Override various paddings set by StoryPromoLi */
+  margin: -${GEL_SPACING_DBL_MINUS_HLF} 0;
```

Comment:
Is there any nicer way we could this by modifying `StoryPromoLi`, maybe give it some configuration to remove the padding?

Reply:
I thought of that initially but there's multiple layers where the padding for `StoryPromoLi` has been overridden already and with varying breakpoints and none of those match the requirements for recommendations.
![Screen Shot 2020-06-25 at 08 49 27](https://user-images.githubusercontent.com/25608386/85678929-d878e880-b6c0-11ea-869b-e14d4b4dcefb.png)

I used !important initially but had to revert to this. I'm open to suggestions if anyone has an idea

Reply:
Unless anyone can think of a neater way to do this now, can we raise a tech debt issue to sort this out? It's probably better to override the margin here than try and fix everything in this one PR, I agree.